pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -164,27 +208,36 @@ configuration in Istio.
 #traceAlwaysSampleErrors=false
-## A comma separated, list of additional custom tag keys to include along as metric tags for the
-## derived RED(Request, Error, Duration) metrics. Applicable to Jaeger and Zipkin integration only.
-#traceDerivedCustomTagKeys=tenant, env, location
 
-## The following settings are used to configure histogram ingestion:
-## Histograms can be ingested in wavefront scalar and distribution format. For scalar samples ports can be specified for
-## minute, hour and day granularity. Granularity for the distribution format is encoded inline.
-## Before using any of these settings, reach out to Wavefront Support to ensure your account is enabled for native Histogram
-## support and to optimize the settings for your specific use case.
+########################################## HISTOGRAM ACCUMULATION SETTINGS #############################################
+## Histograms can be ingested in Wavefront scalar and distribution format. For scalar samples ports can be specified for
+## minute, hour and day granularity. Granularity for the distribution format is encoded inline. Before using any of
+## these settings, reach out to Wavefront Support to ensure your account is enabled for native Histogram support and
+## to optimize the settings for your specific use case.
+
+## Accumulation parameters
+## Directory for persisting proxy state, must be writable.
+#histogramStateDirectory=/var/spool/wavefront-proxy
+## Interval to write back accumulation changes to disk in milliseconds (only applicable when memory cache is enabled).
+#histogramAccumulatorResolveInterval=5000
+## Interval to check for histograms ready to be sent to Wavefront, in milliseconds.
+#histogramAccumulatorFlushInterval=10000
+## Max number of histograms to send to Wavefront in one flush (Default: no limit)
+#histogramAccumulatorFlushMaxBatchSize=4000
+## Whether to persist accumulation state. WARNING any unflushed histograms will be lost on proxy shutdown if disabled.
+#persistAccumulator=true
+## Maximum line length for received histogram data (Default: 65536)
+#histogramMaxReceivedLength=65536
+## Maximum allowed request size (in bytes) for incoming HTTP requests on histogram ports (Default: 16MB)
+#histogramHttpBufferSize=16777216
 
 ## Wavefront format, minute aggregation:
 ## Comma-separated list of ports to listen on.
 #histogramMinuteListenerPorts=40001
-## Number of accumulators per minute port
-#histogramMinuteAccumulators=2
 ## Time-to-live in seconds for a minute granularity accumulation on the proxy (before the intermediary is shipped to WF).
 #histogramMinuteFlushSecs=70
-## Bounds the number of centroids per histogram. Must be in [20;1000], default: 100
-#histogramMinuteCompression=20
-## Average number of bytes in a [UTF-8] encoded histogram key. ~metric, source and tags concatenation.
-#histogramMinuteAvgKeyBytes=150
+## Bounds the number of centroids per histogram. Must be in [20;1000], default: 32
```

Comment:
Will everyone know what "must be in [20;1000]" means? If you think so, just ignore this comment.